pkg/authorization/api/types.go

```diff
@@ -67,6 +69,7 @@ var (
 		KubeExposedGroupName:   {"pods", "replicationcontrollers", "services", "endpoints", "pods/log"},
 		KubeAllGroupName:       {KubeInternalsGroupName, KubeExposedGroupName, QuotaGroupName},
 		KubeStatusGroupName:    {"pods/status", "resourcequotas/status", "namespaces/status"},
+		KubePodAdminGroupName:  {"pod/exec", "pod/proxy", "pod/portforward"},
```

Comment:
Hrm... port forward seems too aggressive to be admin only, as does proxy.  Exec is definitely distinct